src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -555,6 +555,58 @@ GUI_TEST_CLASS_DEFINITION(test_7212) {
     GTUtilsTaskTreeView::waitTaskFinished(os);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7227) {
+    /*
+    1. Open WD
+    2. Add scheme "Read sequence"->"Write sequence"
+    3. Set input file: sample/Genbank/murine.gb
+    4. Set some output file in "UGENE Database" format
+    5. Run
+    Expected state: execution without any errors
+    */
+    GTUtilsWorkflowDesigner::openWorkflowDesigner(os);
+
+    WorkflowProcessItem *readSeq = GTUtilsWorkflowDesigner::addElement(os, "Read Sequence", true);
+    WorkflowProcessItem *writeSeq = GTUtilsWorkflowDesigner::addElement(os, "Write Sequence", true);
+
+    GTUtilsWorkflowDesigner::connect(os, readSeq, writeSeq);
+
+    GTUtilsWorkflowDesigner::addInputFile(os, "Read Sequence", dataDir + "/samples/Genbank/murine.gb");
+    GTUtilsWorkflowDesigner::click(os, "Write Sequence");
+    GTUtilsWorkflowDesigner::setParameter(os, "Document format", "UGENE Database", GTUtilsWorkflowDesigner::comboValue);
+
+    const GTLogTracer logTracer;
```

Comment:
Do not use consts randomly. We either use them everywhere consistently or do not use them at all (at least for locals)